.ideainspect

```diff
@@ -21,3 +15,10 @@ resultdir: target/inspection-results
 
 # IDEA project root directory containing the ".idea" directory
 # rootdir: .
+
+# Library folder containing idea.properties More info at:
+# http://tools.android.com/tech-docs/configuration
+# lib: ~/Library/Preferences/AndroidStudio2.1/
```

Comment:
In IDEA the path is `$IDEA_HOME/bin/idea.properties`. Does this differ in Android Studio?

So instead of obscure autodetection logic I'd rather go for a explicit path to the `idea.properties` file for Android Studio and document it as optional / not required for the default location aside the `idea.sh` script we are already autodetecting.